fetcher.go

```diff
@@ -32,23 +38,28 @@ func (f *Fetcher) Register(keyPrefix string, cbf Callback) bool {
 		return false
 	}
 
-	f.cb[keyPrefix] = cbf
+	expiry := 0
+	cbStruct := CallbackExpiry{cbf, time.Duration(expiry)}
+	f.cb[keyPrefix] = cbStruct
+
 	return true
 }
 
-func (f *Fetcher) Execute(key string) (interface{}, error) {
+func (f *Fetcher) Execute(key string) (interface{}, error, time.Duration) {
```

Comment:
Add method GetCallback() CallbackExpirty return obj instead of 3 different values

Reply:
Register method signature change with time.Duration